spec.html

```diff
@@ -35975,16 +35979,19 @@ <h1>
             1. Let _flags_ be _R_.[[OriginalFlags]].
             1. If _flags_ contains *"g"*, let _global_ be *true*; else let _global_ be *false*.
             1. If _flags_ contains *"y"*, let _sticky_ be *true*; else let _sticky_ be *false*.
+            1. If _flags_ contains *"d"*, let _hasIndices_ be *true*; else let _hasIndices_ be *false*.
             1. If _global_ is *false* and _sticky_ is *false*, set _lastIndex_ to 0.
             1. Let _matcher_ be _R_.[[RegExpMatcher]].
             1. If _flags_ contains *"u"*, let _fullUnicode_ be *true*; else let _fullUnicode_ be *false*.
             1. Let _matchSucceeded_ be *false*.
+            1. If _fullUnicode_ is *true*, let _input_ be ! StringToCodePoints(_S_). Otherwise, let _input_ be a List whose elements are the code units that are the elements of _S_. Each element of _input_ is considered to be a character.
```

Comment:
```suggestion
            1. If _fullUnicode_ is *true*, let _input_ be ! StringToCodePoints(_S_). Otherwise, let _input_ be a List whose elements are the code units that are the elements of _S_.
            1. NOTE: Each element of _input_ is considered to be a character.
```
It's nice to separate the commentary from the 'executable' step. Doing so also makes it a bit clearer that the commentary applies to both possibilities, not just the "Otherwise" part.